cime_config/tests.py

```diff
@@ -69,7 +69,8 @@
             "ERS.ELM_USRDAT.I1850ELM.elm-usrdat",
             "ERS.r05_r05.IELM.elm-V2_ELM_MOSART_features",
             "ERS.f09_f09.IELM.elm-solar_rad",
-            "ERS.f09_f09.IELM.elm-lnd_rof_2way"
+            "ERS.f09_f09.IELM.elm-lnd_rof_2way",
+            "ERS.f09_f09.IELM.elm-snicar_ad_improve"
```

Comment:
How about renaming this test to `ERS.f09_f09.IELM.elm-koch_snowflake`?

Reply:
Agree

Reply:
Done.